tests/publiccloud/slem_prepare.pm

```diff
@@ -23,7 +23,7 @@ sub run {
     if (get_var("PUBLIC_CLOUD_CONTAINERS")) {
         my $runtime = get_required_var('CONTAINER_RUNTIMES');
         # Install packages for container test runs
-        trup_call("pkg install $runtime toolbox");
+        trup_call("pkg install $runtime toolbox cni-plugins");
```

Comment:
shouldn't we drop cni testing when it comes to sle-micro 6.0+?

Reply:
IDK ? without this package results are red: http://quasar.suse.cz/tests/26

Reply:
ok, this is 5.5. So what versions of sle-micro and sl-micro we should run this test against?

By default, cni is not used, but netavark -> http://quasar.suse.cz/tests/26#step/podman/20

Either we have a test bug, that something is mangling the network backend, or there is a product bug

Reply:
Oh